modules/ossm-mt-namespaces.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-mt-ossm.adoc
+
+[id="ossm-mt-namespaces_{context}"]
+= Configuring namespaces in multi-tenant installations
+
+A multi-tenant control plane installation only affects namespaces configured as part of the {ProductShortName}. You must specify the namespaces associated with the {ProductShortName} in a `ServiceMeshMemberRoll` resource located in the same namespace as the `ServiceMeshControlPlane` resource and name it `default`. Here is an example that joins the bookinfo namespace into the {ProductShortName}:
+
+.Prerequisites
+
+* An installed, verified {ProductShortName} operator.
+* Location of the namespace where the custom resource used to install the control plane.
+
+.Procedure
+
+. Create a custom resource file named `ServiceMeshMemberRoll` in the same namespace as the `ServiceMeshControlPlane` custom resource.
+
+. Name the resource `default`.
+
+. Add the namespaces to the member list in the `ServiceMeshMemberRoll`. The `bookinfo` namespace is joined to the {ProductShortName} in this example.
++
+.Namespace configuration example
+
+[source,yaml]
+----
+  apiVersion: maistra.io/v1
+  kind: ServiceMeshMemberRoll
+  metadata:
+    name: default
+  spec:
+    members:
+    # a list of namespaces joined into the service mesh
+    - bookinfo
+----
+
+== Additional multi-tenant namespace information
```

Comment:
Procedures should have only one heading. Please make this content a separate module or present the information in a different way.

Reply:
I rolled this up into the intro, because it is information the reader should be aware of before starting the procedure.